app/src/main/java/io/gnosis/safe/ui/settings/view/AddressItem.kt

```diff
@@ -21,7 +21,7 @@ class AddressItem @JvmOverloads constructor(
     defStyleAttr: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
 
-    private val binding by lazy { ViewAddressItemBinding.inflate(LayoutInflater.from(context), this) }
```

Comment:
What was/is the problem with the lazy initialization? Should we change this in other places, too?

Reply:
No, I have changed it back. Better to have it as lazy, so that inflation happens deferred